Makefile

```diff
@@ -36,6 +36,8 @@ clean:
 CRD_SCHEMA_GEN_APIS := $(shell echo ./vendor/github.com/openshift/api/{authorization/v1,config/v1,quota/v1,security/v1,operator/v1alpha1,console/v1})
 CRD_SCHEMA_GEN_VERSION :=v0.2.1
 
+$(call add-crd-gen,manifests,$(CRD_SCHEMA_GEN_APIS),./manifests,./manifests)
```

Comment:
This should not be needed. With https://github.com/openshift/api/pull/470 in place and it looks like your bump already has that you should re-use the manifests provided by openshift/api. Sync with @damemi I thought he wanted to do it :wink:

Reply:
Until https://github.com/openshift/cluster-config-operator/pull/102 merges those CRDs in openshift/api aren't picked up anywhere, meaning we still need to update here if you want changes pulled in